docs/index.md

```diff
@@ -94,7 +94,7 @@ In the example below, there is an expandable panel that can reveal more content.
 
 A binary tree is a tree data structure in which each node has at most two children, which are referred to as the left child and the right child. Primitive data types on the other hand ...
 
-<panel header="%%{{ icon_info }} Some example primitive data types%%">
+<panel header="%%:fas-info-circle: Some example primitive data types%%">
```

Comment:
Change the definition of the variable in variables.md instead?

Reply:
Applies to other `icon_*` variables too.

Reply:
Currently, the `icon_*` variables have icon fonts of the form `{{ icon_font | safe }}`. I assumed it was a workaround for including them in other variables, in which case the new syntax does not need it anymore.

Did you actually mean for them to be semantic aliases instead? If so, I will add them back in.

Reply:
> Did you actually mean for them to be semantic aliases instead? If so, I will add them back in.

Yes, that was my intention. Easier to remember.

Reply:
@damithc I have added the variables back!